src/osx/cocoa/window.mm

```diff
@@ -3416,21 +3467,21 @@ static NSComparisonResult CocoaLowerWindowCompareFunction(KindOfView first, Kind
     {
         if ( IsUserPane() )
         {
-            // We need to get rid of the old data, if we have it, in any case.
-            delete m_gesturesImpl;
-
             if ( eventsMask == wxTOUCH_NONE )
             {
-                if ( m_gesturesImpl )
+                if ( wxCocoaGesturesImpl::EraseForWindow(this) )
```

Comment:
Suffers from the same freeing issue as has been fixed in GTK: Calling EnableTouchEvents from a PressAndTap Event destroys the wxCocoaGesturesImpl object, while a function call for wxCocoaGesturesImpl  is still active.

Reply:
Sorry, I won't have time to fix this any time soon. If you can make a PR/patch applying the same workaround, I'd be glad to apply it though. TIA!